phpstan.neon.dist

```diff
@@ -0,0 +1,14 @@
+parameters:
+    level: 1
```

Comment:
hmm, level 1, @simPod do you plan in increasing the level with future pull requests?

Reply:
Yup